metis-enrichment/metis-enrichment-common/src/main/java/eu/europeana/enrichment/utils/EntityTypeUtils.java

```diff
@@ -0,0 +1,83 @@
+package eu.europeana.enrichment.utils;
+
+import eu.europeana.corelib.solr.entity.AbstractEdmEntityImpl;
+import eu.europeana.corelib.solr.entity.AgentImpl;
+import eu.europeana.corelib.solr.entity.ConceptImpl;
+import eu.europeana.corelib.solr.entity.OrganizationImpl;
+import eu.europeana.corelib.solr.entity.PlaceImpl;
+import eu.europeana.corelib.solr.entity.TimespanImpl;
+import eu.europeana.enrichment.api.internal.AgentTermList;
+import eu.europeana.enrichment.api.internal.ConceptTermList;
+import eu.europeana.enrichment.api.internal.MongoTermList;
+import eu.europeana.enrichment.api.internal.OrganizationTermList;
+import eu.europeana.enrichment.api.internal.PlaceTermList;
+import eu.europeana.enrichment.api.internal.TimespanTermList;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author Simon Tzanakis
+ * @since 2020-07-17
+ */
+public class EntityTypeUtils {
+
+  private static final List<EntityInfo<?, ?>> ENTITY_INFOS = createEntityTypeList();
+
+  private static List<EntityInfo<?, ?>> createEntityTypeList() {
+    final ArrayList<EntityInfo<?, ?>> entityInfos = new ArrayList<>();
+    entityInfos.add(new EntityInfo<>(EntityType.AGENT, AgentTermList.class, AgentImpl.class));
+    entityInfos.add(new EntityInfo<>(EntityType.CONCEPT, ConceptTermList.class, ConceptImpl.class));
+    entityInfos.add(new EntityInfo<>(EntityType.PLACE, PlaceTermList.class, PlaceImpl.class));
+    entityInfos
+        .add(new EntityInfo<>(EntityType.TIMESPAN, TimespanTermList.class, TimespanImpl.class));
+    entityInfos.add(new EntityInfo<>(EntityType.ORGANIZATION, OrganizationTermList.class,
+        OrganizationImpl.class));
+    return entityInfos;
+  }
+
+  public static EntityInfo<?, ?> getEntityMongoTermListClass(EntityType entityType) {
+    return ENTITY_INFOS.stream().filter((entityInfo) -> entityType == entityInfo.entityType)
+        .findFirst().orElse(null);
+
+  }
+
+  public static EntityType getEntityTypeFromClassImpl(String entityClassImpl) {
+    return ENTITY_INFOS.stream()
+        .filter(entityInfo -> entityInfo.getEntityClassImpl().getSimpleName()
+            .equalsIgnoreCase(entityClassImpl)).findFirst().map(EntityInfo::getEntityType)
+        .orElse(null);
+  }
+  public static EntityInfo<?, ?> getEntityTypeFromClassImplCLASS(String entityClassImpl) {
```

Comment:
According to my IDE, this method is never used. Also, the name is not very good.

Reply:
That IDE... :D 
Weird leftovers, removed!